contrib/inventory/ec2.py

```diff
@@ -672,7 +672,9 @@ def get_elasticache_clusters_by_region(self, region):
                 error = self.get_auth_error_message()
             if not e.reason == "Forbidden":
                 error = "Looks like AWS ElastiCache is down:\n%s" % e.message
-            self.fail_with_error(error, 'getting ElastiCache clusters')
+                self.fail_with_error(error, 'getting ElastiCache clusters')
+            else:
+                return
```

Comment:
disagree.  it is an error to request something you don't have rights for.  you don't know that there aren't any rds instances.
i'd suggest that addressing the situation you're discussing would be better done defaulting `include_rds_clusters` to false in the default ini file.